index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <section id="welcome-section">
+        <h1>Hey I am Mimic</h1>
+    </section>
+    <section id="projects">
+        <p class="project-tile">Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloremque eum, perferendis odio aliquid dolorem dolores officiis, ipsum rerum totam provident aut, nihil ut velit repellat quibusdam incidunt corrupti rem consequuntur.</p>
+        <a href="#">Tribute Page</a>
+        <nav id="navbar">
+            <a href="welcome-section">Welcome</a>
+            <a href="#">Dead Link</a>
+        </nav>
+    </section>
+    <a id="profile-link" href="https://github.com/rhizzen/Botlhale-Frank" target="_blank">GitHub Repo</a>
```

Comment:
- [ ] ... .profile-link also looks like it would be better off in your <footer element. Grouping these links makes your code easier to understand.